packages/blade/src/components/Table/__tests__/Table.web.test.tsx

```diff
@@ -617,6 +617,45 @@ describe('<Table />', () => {
     expect(getAllByRole('row')[0]).toHaveTextContent('completed');
   });
 
+  it('should call onHover when row is hovered', async () => {
+    const onHover = jest.fn();
+    const user = userEvent.setup();
+    const { getByText } = renderWithTheme(
+      <Table data={{ nodes: nodes.slice(0, 5) }} selectionType="single">
+        {(tableData) => (
+          <>
+            <TableHeader>
+              <TableHeaderRow>
+                <TableHeaderCell>Payment ID</TableHeaderCell>
+                <TableHeaderCell>Amount</TableHeaderCell>
+                <TableHeaderCell>Status</TableHeaderCell>
+                <TableHeaderCell>Type</TableHeaderCell>
+                <TableHeaderCell>Method</TableHeaderCell>
+                <TableHeaderCell>Name</TableHeaderCell>
+              </TableHeaderRow>
+            </TableHeader>
+            <TableBody>
+              {tableData.map((tableItem, index) => (
+                <TableRow item={tableItem} key={index} onHover={onHover}>
```

Comment:
should it be `onMouseOver` to make it consistent with web's event handler? that is what we have on Box as well I guess 🤔

Reply:
Also we should use `onMouseEnter` & not `onMouseOver`. onMouseOver will keep retriggering for the same row over and over again whenever the mouse enters any child element. We want it to only trigger once for the whole row

Reply:
> why did we not use onMouseOver for card

No particular, reason. There was no discussion on event names in Interactive Card API [Decisions](https://github.com/razorpay/blade/blob/master/packages/blade/src/components/Card/_decisions/interactive-cards.md). 

For Card maybe onHover makes sense since users should not care about the internal implementation on if it's "onMouseEnter" or "onMouseOver". 

Tho if we want to standardise if further to keep it same with web, I don't see an issue with onMouseEnter also.

Reply:
Calling it onHover since onMouseEnter is a web event and expects callbacks that are true to `React.MouseEventHandler<HTMLElement>` but we only want to pass `{ item }` as a callback